src/main/resources/templates/non-strategic/utIacJrLondonDailyHearingList.html

```diff
@@ -147,7 +147,7 @@ <h2 id="page-heading" class="govuk-heading-l" th:text="${i18n.listName}"/>
       <tbody class="govuk-table__body">
       <tr th:each="hearing : ${data}">
         <td class="govuk-table__cell" th:text="${hearing.get('hearingTime')}"/>
-        <td class="govuk-table__cell" th:text="${hearing.get('applicant')}"/>
+        <td class="govuk-table__cell" th:text="${hearing.get('caseTitle')}"/>
```

Comment:
This schema is shared between London, Cardiff, Manchester and Birmingham. The ticket only mention changing London. Have you confirmed with Kim all the other list types need changing?